boards.cfg

```diff
@@ -69,6 +69,7 @@ isvp_t31al_sfcnor_ddr128M   mips   xburst   isvp_t31    ingenic   t31   isvp_t31
 isvp_t31lc_sfcnor           mips   xburst   isvp_t31    ingenic   t31   isvp_t31:SPL_SFC_SUPPORT,ENV_IS_IN_SPI_FLASH,SPL_SFC_NOR,JZ_MMC_MSC0,T31LC,LITE_VERSION
 isvp_t31_xiaomi_sfcnor      mips   xburst   isvp_t31    ingenic   t31   isvp_t31:SPL_SFC_SUPPORT,ENV_IS_IN_SPI_FLASH,SPL_SFC_NOR,JZ_MMC_MSC0,LITE_VERSION,XIAOMI_SPL
 isvp_t31_xiaomi_alt_sfcnor      mips   xburst   isvp_t31    ingenic   t31   isvp_t31:SPL_SFC_SUPPORT,ENV_IS_IN_SPI_FLASH,SPL_SFC_NOR,JZ_MMC_MSC0,LITE_VERSION,XIAOMI_SPL,XIAOMI_SPL_ALT
+isvp_t31_motorcomm_sfcnor   mips   xburst   isvp_t31    ingenic   t31   isvp_t31:SPL_SFC_SUPPORT,ENV_IS_IN_SPI_FLASH,SPL_SFC_NOR,JZ_MMC_MSC0,MOTORCOMM_YTXXXX
```

Comment:
whats the model of the motorcomm IC chip? lets rename it to that to reflect compatability, instead of XXXX

Reply:
done